gmso/external/convert_hoomd.py

```diff
@@ -1338,6 +1353,21 @@ def _parse_harmonic_improper(
     return container
 
 
+def _parse_periodic_improper(
```

Comment:
So I think this is fine if we used the `periodicimproperpotential` currently in GMSO. However, I think due to the GAFF file we're using, we're fixing `d = 1` in this potential. I just want to make sure others are on board with this decision, because HOOMD can do simulations when d is not fixed to be 1. The other option would be to make a `hoomdperiodicimproperpotential` json in `gmso/lib/jsons`

Reply:
I think we should opt for a more flexible version, i.e., when `d` is not fixed to `1`, and have it set to `1` specifically when parsing GAFF

Reply:
I think this is resolved for now using `.get("d", 1.0)`